forge-gui/res/cardsfolder/upcoming/breathe_your_last.txt

```diff
@@ -0,0 +1,7 @@
+Name:Breathe Your Last
+ManaCost:1 B B
+Types:Instant
+A:SP$ Destroy | ValidTgts$ Creature,Planeswalker | TgtPrompt$ Select target creature or planeswalker | SubAbility$ DBGainLife | SpellDescription$ Destroy target creature or planeswalker. You gain 1 life for each of its colors.
+SVar:DBGainLife:DB$ GainLife | LifeAmount$ X
+SVar:X:Targeted$CardNumColors

```

Comment:
This is good enough, right? Don't need any extra LKI stuff

Reply:
yes